{
  "uid" : "72965d83f9c0c411",
  "name" : "someFailingTestsWithDataProvider",
  "fullName" : "test.java.tests.DummyTest.someFailingTestsWithDataProvider",
  "historyId" : "6f1486d7d0cecb5aa22973477b82de55",
  "time" : {
    "start" : 1605989573879,
    "stop" : 1605989573881,
    "duration" : 2
  },
  "status" : "skipped",
  "statusMessage" : "expected [2] but found [1]",
  "statusTrace" : "java.lang.AssertionError: expected [2] but found [1]\r\n\tat org.testng.Assert.fail(Assert.java:97)\r\n\tat org.testng.Assert.assertEqualsImpl(Assert.java:136)\r\n\tat org.testng.Assert.assertEquals(Assert.java:118)\r\n\tat org.testng.Assert.assertEquals(Assert.java:839)\r\n\tat org.testng.Assert.assertEquals(Assert.java:849)\r\n\tat test.java.tests.DummyTest.someFailingTestsWithDataProvider(DummyTest.java:29)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\r\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)\r\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\r\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:564)\r\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\r\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)\r\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\r\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)\r\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)\r\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\r\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)\r\n\tat java.base/java.lang.Thread.run(Thread.java:832)\r\n",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ {
    "name" : "setUp",
    "time" : {
      "start" : 1605989566856,
      "stop" : 1605989573873,
      "duration" : 7017
    },
    "description" : "",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  } ],
  "afterStages" : [ {
    "name" : "tearDown",
    "time" : {
      "start" : 1605989573883,
      "stop" : 1605989574547,
      "duration" : 664
    },
    "description" : "",
    "status" : "passed",
    "steps" : [ ],
    "attachments" : [ ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "hasContent" : false,
    "attachmentsCount" : 0,
    "shouldDisplayMessage" : false
  } ],
  "labels" : [ {
    "name" : "package",
    "value" : "test.java.tests.DummyTest"
  }, {
    "name" : "testClass",
    "value" : "test.java.tests.DummyTest"
  }, {
    "name" : "testMethod",
    "value" : "someFailingTestsWithDataProvider"
  }, {
    "name" : "parentSuite",
    "value" : "Lesson 12"
  }, {
    "name" : "suite",
    "value" : "Three tests"
  }, {
    "name" : "subSuite",
    "value" : "test.java.tests.DummyTest"
  }, {
    "name" : "host",
    "value" : "XPS7"
  }, {
    "name" : "thread",
    "value" : "31136@XPS7.TestNG-test=Three tests-1(17)"
  }, {
    "name" : "framework",
    "value" : "testng"
  }, {
    "name" : "language",
    "value" : "java"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "arg0",
    "value" : "1"
  } ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "72965d83f9c0c411.json",
  "parameterValues" : [ "1" ]
}